src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -140,10 +140,18 @@ public void InitializeService(ServiceHost serviceHost)
                     Version = Version.Parse(serverInfo.ServerVersion),
                     Platform = hostInfo.Platform
                 };
-
                 var db = SqlAssessmentService.GetDatabaseLocator(server, connection.Database);
-                var connectionString = ConnectionService.BuildConnectionString(connInfo.ConnectionDetails);
-                var results = await GetAssessmentItems(server, connectionString);
+                var connectionStrings = new List<string>();
+                if (parameters.Databases != null) 
```

Comment:
This is changing behavior - before it would pass in the connection string for the original db and use that. Is the intention that the callers are now required to pass in the list of dbs to use? What happens if they pass in an empty array?

Also am I missing something or is the db object above not actually used anywhere?

Reply:
Yes, we now have the user select the DBs that will be assessed, so those DBs will get passed in through this parameter.  I use the parameters.Databases in line 147 to loop through each DB and build a connection string for each DB.